yt_dlp/extractor/arte.py

```diff
@@ -46,154 +49,150 @@ class ArteTVIE(ArteTVBaseIE):
         'only_matching': True,
     }]
 
+    _GEO_BYPASS = True
+
+    __LANG_MAP = {
```

Comment:
This seemed like a very generic name, one that potentially might be used by some base class, so I decided to mitigate any potential naming conflicts, just in case (lol no `private`)

I might have used a module-level constant instead.